main.go

```diff
@@ -204,10 +204,11 @@ func main() {
 	eventBroadcaster.StartLogging(klog.V(log.DEBUG).Infof)
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "submariner-controller"})
 
-	startLeaderElection(leClient, recorder, start)
+	startLeaderElection(leClient, recorder, start, cableEngineSyncer.CleanupOwnEntry)
```

Comment:
the new "end" function will not force an exit of the process, we need to do that.